### ISSUE TITLE
i am seeing the below error while apply

### ISSUE THREAD

**Issue by manjukashyap** [oracle-terraform-modules/terraform-oci-vcn#10]
Error: did not find a proper configuration for private key

  on vcn.tf line 4, in resource "oci_core_vcn" "vcn":
   4: resource "oci_core_vcn" "vcn" {

I have the keys placed in the corresponding ../keys/***.pem

Please assist.

terraform version: 0.13.4.

**Comment by hyder** [comment]
Your oci provider doesn't seem to have been properly configured. See https://github.com/oracle-terraform-modules/terraform-oci-vcn/blob/master/docs/quickstart.adoc

**Comment by RahulMetangale** [comment]
Even i am getting same error. I created the configuration by running the "oci setup config" command and also uploaded the public key on portal. I am able to run following but when i run the example give for this module i get the error "did not find the configuration for private key"
```
provider "oci" {
  fingerprint          = var.api_fingerprint
  private_key_path     = var.api_private_key_path
  region               = var.region
  tenancy_ocid         = var.tenancy_id
  user_ocid            = var.user_id
}

# Get a list of Availability Domains
data "oci_identity_availability_domains" "ads" {
  compartment_id = var.tenancy_id
}

# Output the result
output "show-ads" {
  value = "data.oci_identity_availability_domains.ads.availability_domains"
}
```

**Comment by RahulMetangale** [comment]
I was able to to fix the issue.  Following helped in resolving the issue:
- in my case i was trying to pass some values from command line and some from tfvars file. We noticed that compartment_id was pickup from tfvars file instead of command line. Since i wanted to pass the value from command line i removed the  compartment_id from tfvars file. I my case file was the last parameter. I also removed other parameters that i wanted to pass from command line from tvvars file
Here is the command i was using
`terraform apply -var tenancy_id="" -var user_id=""  -var api_fingerprint="" -var api_private_key_path=""  -var compartment_id="" -var-file=terraform.tfvars.example`

- In main.tf file since i wanted to accept compartment id from command line i change compartment_id = "" to compartment_id=var.compartment_id
-Added provider.tf file with following code
```
provider "oci" {
  fingerprint          = var.api_fingerprint
  private_key_path     = var.api_private_key_path
  region               = var.region
  tenancy_ocid         = var.tenancy_id
  user_ocid            = var.user_id
}
```



-
